relationships/synthesis/INFRA-KUBERNETES_CONTAINER-to-EXT-SERVICE.yml

```diff
@@ -0,0 +1,33 @@
+relationships:
+  - name: k8sContainerEXTService
+    version: "1"
+    origins:
+      - OpenTelemetry
+    conditions:
+      - attribute: eventType
+        anyOf: [ "Metric" ]
+      - attribute: entity.type
+        anyOf: [ "CONTAINER" ]
```

Comment:
Here + line 21

I think this is trying to relate INFRA-CONTAINER to INFRA-CONTAINERS ?